drivers/storage/portworx/component/csi.go

```diff
@@ -666,6 +668,10 @@ func getCSIDeploymentSpec(
 		},
 	}
 
+	if pxutil.IsStorkEnabled(cluster) {
```

Comment:
This won't handle if stork is disabled at runtime. This is not a common scenario, but we should handle it.
Currently the CSI deployment gets recreated only if certain things change in the deployment based on the storagecluster spec. Check line https://github.com/libopenstorage/operator/pull/807/files#diff-41fa95b9f2df1987f95180171e28b9d77de07ce97cf8d5c34dea99ca3c54e0c2R506

We will have to add some custom logic there for the scheduler field or check with @haibinxie to use the custom DeepCopy function he introduced.

Reply:
Can you verify what happens if the stork scheduler is removed at runtime? Try deleting one of the stork pods and see what happens.

Reply:
Ah yes, we need to make sure the px-csi-ext is bounced when stork is disabled or enabled.

Reply:
@piyush-nimbalkar Added the changes to handle the scenario when stork is enabled or disabled at run time.